### PR TITLE
Quick start front

### PR DESCRIPTION
**Pull request by k2shah**
moved the quick start guide and related assets to the top leave. Minor clean up elsewhere for consistency and added another video and ref

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -36,16 +41,22 @@ QuadsManip_ repository.
...
-Features
-========
 
-- .. TODO: List our current features here
+Getting Started
+---------------
 
-Installation
-============
+Please review our :doc:`quickstart/index` to get starte.
```
Comment (exoticDFT) [inline docs/source/index.rst]: started*

**Review by exoticDFT — CHANGES_REQUESTED** [review]
@k2shah The restructuring looks great! I really think this makes the documentation flow really well and very professional. Only one request.

The formatting of some of the files exceeds the character limit. I'm not concerned about the links that run too long, etc. But for the paragraphs, lists, etc. we should keep to the limit as much as possible.

File: docs/source/index.rst (hunk 1 above)
Comment (k2shah) [reply docs/source/index.rst]: fixed

**Comment by k2shah** [comment]
Ya i was going to set up an autoformater but in a different PR. i'll make a issue for setting it up

**Review by exoticDFT — APPROVED** [review]
Looks great!

**Comment by rhaksar** [comment]
Comments:
- on the "Ground Station" bullet point under "Setup", use an oxford comma: "mocap.local[,] and your own personal computer"
- on the "Companion Computer" bullet point, maybe clarify that this computer is typically on-board the vehicle?
- Use "ol'" or "old" instead of "ole" under the Safety Checklist
- does "4" need to be italic under "Mechanical"?
- under the "Safety Checklist" maybe use the same shorthand as the "Setup"? so "**FCU**" instead of "fcu" and "**CC**" instead of "cc"

**Pull request merged**